apps/forms/41-control-value-accessor/src/app/rating-control/rating-control.component.html

```diff
@@ -5,7 +5,7 @@
   </symbol>
 </svg>
 <div class="rating">
-  @for (item of [].constructor(5); track item) {
+  @for (item of [].constructor(5); track $index) {
```

Comment:
This is a fix for console warnings: 
![image](https://github.com/tomalaforge/angular-challenges/assets/15015378/f58dc095-15ed-4c9e-b574-ed1bc19328bf)